README.md

```diff
@@ -263,6 +263,10 @@ By default Jest ignores everything in `node_modules`. This setting prevents Jest
   "moduleDirectories": ["node_modules", "<path_to_your_sources>"]
 }
 ```
+### TS compiler && error reporting
+- Only syntactic errors are reported by the [tsc `transpileModule`](https://github.com/Microsoft/TypeScript/issues/4864#issuecomment-141567247)
```

Comment:
I think this is a little too technical. Perhaps the first line could be 'ts-jest only returns syntax errors from TSC' or something like that?